scripts/list_boards.py

```diff
@@ -91,7 +91,7 @@ def from_soc(soc, variants):
 @dataclass(frozen=True)
 class Board:
     name: str
-    dir: Path
+    dir: List[Path]
```

Comment:
nit: Maybe `directories`, both to help clarify that this is a list and to align with CMake.

Reply:
Good suggestion, applied.